src/context/Video/getVideos.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+export const getAllVideos = async () => {
+    try {
+      const res = await axios.get("/api/videos");
+      if (res.status === 200) {
+        videoDispatch({type: "SET_VIDEOS", payload: res.data.videos});
+      }
+    } catch(error){
+      console.error("Error in setting videos", error);
```

Comment:
You can add a toast here to show error to user.

Reply:
thanks for pointing it out, I have made the required changes.